certbot/client.py

```diff
@@ -565,24 +571,34 @@ def view_config_changes(config, num=None):
     rev.recovery_routine()
     rev.view_config_changes(num)
 
+def _open_pem_file(cli_arg_path, pem_path):
+    """Open a pem file.
+
+    If cli_arg_path was set by the client, open that.
+    Otherwise, uniquify the file path.
+
+    :param str cli_arg_path: the cli arg name, e.g. cert_path
+    :param str pem_path: the pem file path to open
 
-def _save_chain(chain_pem, chain_path):
+    :returns a file object
```

Comment:
nit: missing colon after `:returns` causing documentation that looks like this:

![screenshot from 2016-05-24 18-53-00](https://cloud.githubusercontent.com/assets/6504915/15525649/c2ae3b6e-21e0-11e6-8132-afcafacc15e8.png)